test/NinjaTools.FluentMockServer.API.Exclude.Tests/Downstream/RequestMapperSpec.cs

```diff
@@ -295,30 +280,20 @@ private void GivenTheInputRequestHasNoHeaders()
 
         private void GivenTheInputRequestHasPath([CanBeNull] string path)
         {
-            if (path != null)
-            {
-                _inputRequest.Path = path;
-            }
+            if (path != null) _inputRequest.Path = path;
         }
 
         private void GivenTheInputRequestHasQueryString([CanBeNull] string querystring)
         {
-            if (querystring != null)
-            {
-                _inputRequest.QueryString = new QueryString(querystring);
-            }
+            if (querystring != null) _inputRequest.QueryString = new QueryString(querystring);
         }
 
 
         private async Task WhenMapped()
         {
             _result = await _requestMapper.MapAsync(_inputRequest);
 
-            if (_result.Data != null && _result.Data is { } data )
-            {
-                _mail = await data.Content.DeserializeAsync<Email>();
-               
-            }
+            if (_result.Data != null && _result.Data is { } data) _mail = await data.Content.DeserializeAsync<Email>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4739715)